build_info.json

```diff
@@ -24,6 +24,14 @@
         ]
       }
     },
+    "operator-race": {
+      "patch": {
```

Comment:
we will run the same set of tests in staging as in patch, so the staging part is also needed. Please use
```"repository": "268558157000.dkr.ecr.us-east-1.amazonaws.com/staging/mongodb-kubernetes"```

Reply:
Have we already decided on a final name for the staging repo ? 
We might want to be more specific than just `staging` 
Like `mongodb-kubernetes-staging`

(this can be addressed in the TD/another PR, I'm just curious)

Reply:
I wanted to keep it similar as we have for `dev`, so short name `staging` is ok. I will add it to TD.